metatheory/TestDetailed.hs

```diff
@@ -44,93 +44,24 @@ compareResult eq mode test = do
         ExitSuccess   -> return ())
   return $ Finished $ if eq (C.pack plcOutput) (C.pack plcAgdaOutput) then Pass else Fail $ "plc: '" ++ plcOutput ++ "' " ++ "plc-agda: '" ++ plcAgdaOutput ++ "'"
 
-compareResult' :: (C.ByteString -> C.ByteString -> Bool) -> String -> IO Progress
-compareResult' eq test = do
+compareResultMode :: String -> String -> (C.ByteString -> C.ByteString -> Bool) -> String -> IO Progress
```

Comment:
I do slightly wonder if there's a point having these tests that call out to processes. As you do in the NEAT tests, you can just call the underlying functions, and you've got access to both the Haskell and the Agda versions. Am I missing something?

(Doesn't have to be in this PR, just an observation)

Reply:
Good point. I might look into when I do the refactor of `plc-agda`.